[CI] GenerativeIT test failing

**Build Scans:**
- [elasticsearch-periodic #10881 / openjdk24_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/uv2kgcgmcaeem)
- [elasticsearch-pull-request #95338 / part-3](https://gradle-enterprise.elastic.co/s/ebckungjfhqzq)
- [elasticsearch-pull-request #95333 / part-3](https://gradle-enterprise.elastic.co/s/egbs6nrengt2a)
- [elasticsearch-pull-request #95323 / part-3](https://gradle-enterprise.elastic.co/s/tp5nxtn7dsrm4)
- [elasticsearch-pull-request #95269 / part-3](https://gradle-enterprise.elastic.co/s/7puzajar67f7s)
- [elasticsearch-pull-request #95228 / part-3](https://gradle-enterprise.elastic.co/s/nbrsgadlgacfu)
- [elasticsearch-pull-request #95139 / part-3](https://gradle-enterprise.elastic.co/s/yrzo3r5lapntc)
- [elasticsearch-pull-request #94810 / part-3](https://gradle-enterprise.elastic.co/s/4aa2vw56hsibg)
- [elasticsearch-pull-request #94785 / part-3](https://gradle-enterprise.elastic.co/s/s6l2lvi7nccq4)
- [elasticsearch-pull-request #94730 / part-3](https://gradle-enterprise.elastic.co/s/uvvnweukreab6)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT.test" -Dtests.seed=AA9A6A6B3863E1DE -Dtests.locale=lag-Latn-TZ -Dtests.timezone=America/Detroit -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: from client_cidr* | SAMPLE 0.19568499805794204| rename env as message | lookup join message_types_lookup on message | limit 6070 | limit 10313 | dissect message "%{message} %{client_cidr}" | eval  `type` = null, `client_cidr` = false | sort message DESC NULLS LAST | grok message "%{WORD:message}" | limit 14642 | SAMPLE 0.43909259644136533 | SAMPLE 0.8497606742275678 | where  NOT false| rename message as message | lookup join message_types_lookup on message | FORK  (where  NOT false) (where  NOT false | grok message "%{WORD:type} %{WORD:message}"  | limit 11332) (where true) | WHERE _fork == "fork1" | DROP _fork | keep `message`, message, `client_cidr`
exception: method [POST], host [http://[::1]:42245], URI [/_query?format=json&pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
{
  "error" : {
    "root_cause" : [
      {
        "type" : "verification_exception",
        "reason" : "Found 1 problem\nline -1:-1: Column [type] has conf
[truncated]
```

**Issue Reasons:**
- [main] 12 failures in test test (1.2% fail rate in 975 executions)
- [main] 11 failures in step part-3 (2.6% fail rate in 421 executions)
- [main] 11 failures in pipeline elasticsearch-pull-request (2.7% fail rate in 415 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -588,9 +588,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecIT
   method: test {csv-spec:spatial_shapes.ConvertCartesianShapeFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/135455
-- class: org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/134407
 - class: org.elasticsearch.cluster.routing.allocation.decider.RestoreInProgressAllocationDeciderTests
   method: testCanAllocatePrimaryExistingInRestoreInProgress
   issue: https://github.com/elastic/elasticsearch/issues/135566
@@ -603,9 +600,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackIT
   method: testAggTooManyMvLongs
   issue: https://github.com/elastic/elasticsearch/issues/135585
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/134407
 
 # Examples:
 #

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/generator/EsqlQueryGenerator.java
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/generator/EsqlQueryGenerator.java
@@ -21,7 +21,6 @@ import org.elasticsearch.xpack.esql.generator.command.pipe.LimitGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.LookupJoinGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.MvExpandGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.RenameGenerator;
-import org.elasticsearch.xpack.esql.generator.command.pipe.SampleGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.SortGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.StatsGenerator;
 import org.elasticsearch.xpack.esql.generator.command.pipe.TimeSeriesStatsGenerator;
@@ -72,7 +71,8 @@ public class EsqlQueryGenerator {
         LookupJoinGenerator.INSTANCE,
         MvExpandGenerator.INSTANCE,
         RenameGenerator.INSTANCE,
-        SampleGenerator.INSTANCE,
+        // awaits fix for https://github.com/elastic/elasticsearch/issues/135336
+        // SampleGenerator.INSTANCE,
         SortGenerator.INSTANCE,
         StatsGenerator.INSTANCE,
         WhereGenerator.INSTANCE

